Update configure-client-management-azure.adoc (#220)
mulesoft/docs-access-management: modules/ROOT/pages/configure-client-management-azure.adoc

For an example of the `PATCH` payload, see the https://docs.microsoft.com/en-us/graph/api/application-update?view=graph-rest-1.0&tabs=http[Update application] documentation. For an example of the `DELETE` payload, see the https://docs.microsoft.com/en-us/graph/api/application-delete?view=graph-rest-1.0&tabs=http[Delete application] documentation. 
. Click *Create*.

== Microsoft Identity Platform Support

=== Access Tokens 

> For your APIs, Azure AD client management in Anypoint Platform supports both https://docs.microsoft.com/en-us/azure/active-directory/develop/access-tokens[v1 and v2 JSON Web Tokens](JWTs) on Microsoft identity platform. 

Reviewer: ```suggestion

For your APIs, Azure AD client management in Anypoint Platform supports both https://docs.microsoft.com/en-us/azure/active-directory/develop/access-tokens[v1 and v2 JSON Web Tokens](JWTs) on Microsoft identity platform. 
```

=== OAuth 2.0 Endpoints

> Anypoint Platform supports only V2 Azure AD endpoints, i.e. Microsoft Identity Platform endpoints. Take token endpiont for example, Anypoint Platform supports https://login.microsoftonline.com/{tenantId}/oauth2/v2.0/token. 

Reviewer: ```suggestion

Anypoint Platform supports only V2 Azure AD endpoints, such as Microsoft Identity Platform endpoints. Take token endpoint for example, Anypoint Platform supports https://login.microsoftonline.com/{tenantId}/oauth2/v2.0/token. 
```

Author: Thanks for the suggesting!
Can we keep "i.e." or "that is" because "V2 Azure AD" is "Microsoft Identity Platform" (V2 Azure AD endpoints" is "Microsoft Identity platform endpoints"? 

Btw, should we say "Azure AD V2" instead of "V2 Azure Ad"?

Reviewer: ```suggestion
Anypoint Platform supports only tokens obtained using Azure AD v2.0 endpoints, also known as Microsoft identity platform endpoints. For example, Anypoint Platform supports the token endpoint: `https://login.microsoftonline.com/{tenantId}/oauth2/v2.0/token`. 
```
